docs/README.md

```diff
@@ -30,6 +30,20 @@ For configuration examples for the caddy plugin look at our [examples section](/
 ### Location
 The TXT record must be accessible under the subdomain "\_redirect".
 
+### URLs
+* Please encode all URLs in the TXT record since the parser expects encoded URLs. In some cases, unencoded characters may break your configuration.
+* Most notably, ";" should be escaped as "%3B" in a URL since the parser will split the URL on the unencoded semicolon.
+
+* Examples:  
+    ; -> %3B  
+    ? -> %3F  
+    = -> %3D  
+`https://example.com/over/there?name=ferret -> https://example.com/over/there%3Fname%3Dferret`
```

Comment:
This example url might be true or might not be.
Query strings are usually not encoded, if they are they are not query strings and are part of the url. This is a really special case and doesn't feel like the best example.

Better to use something like:
`https://example.com/already/encoded? -> https://example.com/already/encoded%3F`

Reply:
Thanks for pointing that out - I'll change the example we use